src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -3,14 +3,17 @@
 
 using System;
 using System.Collections.Generic;
+#if FEATURE_XML_SCHEMA
```

Comment:
Can you move this down to the lower #if/dev (and keep the extra newlin?

Reply:
Done